config/services.php

```diff
@@ -35,4 +35,16 @@
         'secret' => env('STRIPE_SECRET'),
     ],
 
+    'facebook' => [
```

Comment:
[rank A] I think these social key should be define in .env file. In this constant you can get it from env
And if you put it to env file, please update .env.sample file too